float32 shoulderV
float32 shoulderH
float32 elbow
float32 wristV
float32 wristH
